addons/mail/static/src/models/composer_view.js

```diff
@@ -299,6 +299,51 @@ registerModel({
                 markEventHandled(ev, 'Composer.closeEmojisPopover');
             }
         },
+        /**
+         * @param {KeyboardEvent} ev
+         */
+        onKeydownTextareaEnter(ev) {
```

Comment:
Isn't the name of the handler a bit weird? I would have removed the "Enter".

Reply:
This function is just the "Enter" part of the "onKeydownTextarea" handler.
I cannot yet move the `onKeydownTextarea` to model, because it requires moving `onKeydownTextareaEnter` beforehand.

This split of handler was motivated by long functions.